eth2/beacon/state_machines/base.py

```diff
@@ -72,7 +72,9 @@ def state_transition(self) -> BaseStateTransition:
     # Import block API
     #
     @abstractmethod
-    def import_block(self, block: BaseBeaconBlock) -> Tuple[BeaconState, BaseBeaconBlock]:
+    def import_block(self,
+                     block: BaseBeaconBlock,
+                     is_proposer_mode: bool=False) -> Tuple[BeaconState, BaseBeaconBlock]:
```

Comment:
```suggestion
                     check_proposer_signature: bool=False) -> Tuple[BeaconState, BaseBeaconBlock]:
```

Just a suggestion to make it more explicit what's going on.